build_helpers/buildNPMInternals.sh

```diff
@@ -5,12 +5,12 @@
 var glob = require('glob');
 var path = require('path');
 var fs = require('fs');
+var rimraf = require('rimraf');
```

Comment:
The node file system (`fs`) package doesn't seem to have a dedicated API to remove non-empty directories (see [SO entry](https://stackoverflow.com/questions/18052762/remove-directory-which-is-not-empty)).

So let's add [`rimraf`](https://www.npmjs.com/package/rimraf) as a dev package which should do the same thing as `rm -rf`.

Reply:
Could we have legal ticket?